api/proto/config/devcommon.proto

```diff
@@ -49,4 +49,9 @@ message ConfigItem {
 message Adapter {
   org.lfedge.eve.common.PhyIoType type = 1;
   string name = 2;	// Short hand name such as "com" from bundle
+  EthVF eth_vf = 3;
+}
+message EthVF {
+  string mac = 1;
+  uint32 vlan_id = 2;
 }
```

Comment:
Some comments would be helpful here.
I assume we need the EthVF since the name in the Adapter would be just "eth0". Is that correct?

Reply:
Plus a comment that the EthVF is only relevant if the type is PHY_IO_TYPE_PHY_IO_NET_ETH_VF

Reply:
That is correct